.travis.yml

```diff
@@ -6,7 +6,6 @@ branches:
 python:
  - "3.6"
  - "3.5"
- - "3.4"
```

Comment:
Dropped 3.4 support because FEC is using 3.5, CALC is using 3.6, and I *think* those are the only two projects using this package right now.  (I wanted to be able to use mypy without having to use comments to annotate types, and besides, this simplifies our testing matrix.)